apps/angular-example-counter/src/app/counter/counter.component.html

```diff
@@ -1,9 +1,9 @@
 <div class="App">
   <h1>Counter</h1>
 
-  <div>count: {{ store?.count$ | async }}</div>
-  <div>local: {{ store?.localCount$ | async }}</div>
+  <div>count: {{ store?.store.count$ | async }}</div>
+  <div>local: {{ localCount$ | async }}</div>
 
-  <button (click)="store.counterChange$.next(1)">add</button>
-  <button (click)="store.counterChange$.next(-1)">subtract</button>
+  <button (click)="store.store.counterChange$.next(1)">add</button>
+  <button (click)="store.store.counterChange$.next(-1)">subtract</button>
```

Comment:
this is one thing i don't like that I will address with the angular library we've yet to create.

Reply:
Why optional chain on some lines but not others?